assets/data-labeling/environments/data-labeling-sam/context/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu117-py38-torch1131:biweekly.202305.3
```

Comment:
use a template to resolve latest base image. base image from May will result a sev2 for your team the moment you release your environment

Reply:
See https://github.com/Azure/azureml-assets/wiki/Environments#template-tags-1